core/node/rules/can_add_event.go

```diff
@@ -1048,6 +1076,56 @@ func (params *aeParams) channelMessageReactEntitlements() (*auth.ChainAuthArgs,
 	return chainAuthArgs, nil
 }
 
+func (params *aeParams) updateAutojoinMessageChannelModifyRequirements() (*auth.ChainAuthArgs, error) {
+	userId, err := shared.AddressHex(params.parsedEvent.Event.CreatorAddress)
+	if err != nil {
+		return nil, err
+	}
+
+	inception, err := params.streamView.(events.SpaceStreamView).GetSpaceInception()
+	if err != nil {
+		return nil, err
+	}
+
+	spaceId, err := shared.StreamIdFromBytes(inception.StreamId)
+	if err != nil {
+		return nil, err
+	}
+
+	chainAuthArgs := auth.NewChainAuthArgsForSpace(
+		spaceId,
+		userId,
+		auth.PermissionAddRemoveChannels,
+	)
+
+	return chainAuthArgs, nil
+}
+
+func (params *aeParams) updateChannelShowUserJoinLeaveEventsRequirements() (*auth.ChainAuthArgs, error) {
+	userId, err := shared.AddressHex(params.parsedEvent.Event.CreatorAddress)
+	if err != nil {
+		return nil, err
+	}
+
+	inception, err := params.streamView.(events.SpaceStreamView).GetSpaceInception()
+	if err != nil {
+		return nil, err
+	}
+
+	spaceId, err := shared.StreamIdFromBytes(inception.StreamId)
+	if err != nil {
+		return nil, err
+	}
+
+	chainAuthArgs := auth.NewChainAuthArgsForSpace(
+		spaceId,
+		userId,
+		auth.PermissionAddRemoveChannels,
+	)
+
+	return chainAuthArgs, nil
```

Comment:
the existing function that also uses add remove channels is much smaller, can you follow that convention? `getCreateSpaceChainAuth`